### PR TITLE
feat: Handle properly apns-push-type by adding a push type textfield (SDKCF-6629)

### PR DESCRIPTION
**Pull request by ra-malik-alayli**
# Description
Handle properly `apns-push-type` by adding a push type textfield.

## UI
- Remove the LiveActivity checkbox
- Add a push type textfield with placeholder "Enter APNs push type"

## Behaviour
Pass the push type textfield value to APNs Pusher.

## Links
SDKCF-6629

# Checklist
- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have added to the [changelog](../blob/master/CHANGELOG.md#Unreleased)
- [x] I wrote/updated tests for new/changed code
- [ ] I removed all sensitive data **before every commit**, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard [hunk 1]
```diff
@@ -11,24 +11,24 @@
             <objects>
                 <viewController storyboardIdentifier="PusherViewController" id="XfG-lQ-9wD" customClass="PusherViewController" customModule="PusherMainView" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" translatesAutoresizingMaskIntoConstraints="NO" id="m2S-Jp-Qdl">
```
Comment (ra-malik-alayli) [inline PusherMainView/PusherMainView/Base.lproj/Pusher.storyboard:13]: Note: These value are auto-generated by Interface builder after removing live activity checkbox and adding the push type textfield.

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-gem "fastlane"
+gem "fastlane", "2.201.0"
```
Comment (ra-malik-alayli) [inline Gemfile:3]: Note: `fastlane ci` fails if the minimum version of `fastlane` compatible with`xcodebuild_formatter` is not set.

Comment (imaginaris) [reply Gemfile:3]: Can't we use `>= 2.201.0` or `~> 2.201` rule? Does it have to be exactly 2.201.0?

**Review by imaginaris — COMMENTED** [review]
@ra-malik-alayli Is there a need to update the screenshots?

File: Gemfile (hunk 2 above)
Comment (ra-malik-alayli) [reply Gemfile:3]: @imaginaris I tested it with `2.201.0` and it works with this exact version. Is it needed to be `~> 2.201`?

File: APNS/APNS/URLRequest+Extensions.swift [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+enum HeaderFieldConstants {
+    static let apnsPushType = "apns-push-type"
+}
+
+public enum APNsPushType {
+    public static let liveactivity = "liveactivity"
```
[the remark this answers is not in the document]
Comment (ra-malik-alayli) [reply APNS/APNS/URLRequest+Extensions.swift]: @imaginaris Yes. Done.

**Comment by ra-malik-alayli** [comment]
> @ra-malik-alayli Is there a need to update the screenshots?

@imaginaris I updated screenshots.

File: Gemfile (hunk 2 above)
Comment (imaginaris) [reply Gemfile:3]: Ok maybe that's safer approach. Fastlane is known to introduce major bugs or breaking changes in minor versions

**Review by imaginaris — APPROVED** [review]

**Pull request merged**